client/src/apiClient/WarmFuzzyClient.js

```diff
@@ -0,0 +1,40 @@
+import AbstractClient from "./AbstractClient";
+
+class WarmFuzzyClient extends AbstractClient {
+  // eslint-disable-next-line class-methods-use-this
+  async postFuzzy({ title, description, elevatorId, recipientId }) {
+    const resp = await WarmFuzzyClient.post("/warm-fuzzies", { title, description, elevatorId, recipientId }).catch((error) => {
```

Comment:
is autoformat on save set in vscode? I think this line should wrap.

Reply:
It appears to be set - can you clarify what you're looking for?